.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
```

Comment:
I'm guessing this won't run on this repository until it's actually been merged initially.
You can see the CI run on my fork though: https://github.com/Dreamsorcerer/bthidhub/commits/master